docs/admin/install/serving/install-serving-with-yaml.md

```diff
@@ -378,17 +383,17 @@ The tabs below expand to show instructions for installing each Serving extension
 === "TLS wildcard support"
 
     If you are using a Certificate implementation that supports provisioning
-    wildcard certificates (e.g. cert-manager with a DNS01 issuer), then the most
+    wildcard certificates (for example, cert-manager with a DNS01 issuer), then the most
     efficient way to provision certificates is with the namespace wildcard
-    certificate controller. The following command will install the components needed
-    to provision wildcard certificates in each namespace:
+    certificate controller. Install the components needed to provision wildcard certificates in each
+    namespace by running the command:
 
     ```bash
     kubectl apply -f {{ artifact(repo="serving",file="serving-nscert.yaml")}}
     ```
 
     !!! warning
-        Note this will not work with HTTP01 either via cert-manager or the net-http01
+        Note this doesn't work with HTTP01 either through cert-manager or the net-http01
         options.
```

Comment:
```suggestion
        TLS wildcard support does not work with HTTP01.
```

I'd check if this is accurate w/ someone for eng, @nak3 maybe; in terms of wording though, if this is what it means, I don't think it needs to list all the ways it doesn't work with HTTP01, just that it doesn't work period?

Reply:
Is the following statement true, @nak3?

"TLS wildcard support does not work with HTTP01."

Reply:
Yes, I think it is still true. cc @ZhiminXiang who is a specialist for the auto TLS.

Reply:
Yes this is true. The rewording sounds good to me.